airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/avro_parser.py

```diff
@@ -157,5 +157,11 @@ def _to_output_value(avro_format: AvroFormat, record_type: Mapping[str, Any], re
             return record_value
         if record_type.get("logicalType") == "uuid":
             return uuid.UUID(bytes=record_value)
+        elif record_type.get("logicalType") == "decimal":
+            return str(record_value)
+        elif record_type.get("logicalType") == "local-timestamp-millis":
```

Comment:
I tried reading an avro file with a timestamp field I created locally and the field was read as a datetime.datetime

Reply:
this seems to be different from the avro docs https://avro.apache.org/docs/1.8.0/spec.html#Date